mediathread/djangosherd/apiviews.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import status
+from rest_framework.generics import CreateAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from mediathread.djangosherd.serializers import SherdNoteSerializer
+from mediathread.sequence.apiviews import CsrfExemptSessionAuthentication
+
+
+class SherdNoteCreate(CreateAPIView):
+    authentication_classes = [CsrfExemptSessionAuthentication]
+    permission_classes = [IsAuthenticated]
```

Comment:
Additional permissions are needed to ensure the user can only create an annotation on an asset that is visible to them. We need a check that the asset is within a course where the user is a member. And, that the asset can be edited based on the course permissions.

Reply:
Thanks for listing these out! I will add these tomorrow.